src/widgets/SimpleLabel.js

```diff
@@ -80,6 +86,8 @@ SimpleLabel.defaultProps = {
   label: '',
   labelAlign: 'left',
   textAlign: 'left',
+  labelBackground: 'transparent',
+  textBackground: 'transparent',
```

Comment:
I think we should just use `PageInfo` rather than simple label here, just like other pages?

![image](https://user-images.githubusercontent.com/35858975/73925420-e6f17e00-4932-11ea-851a-326d55f237bc.png)


Also since we may need to show multiple balances i.e. for each payment types, it'll be nice using `PageInfo` I think